doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -442,7 +442,10 @@ the conversion in hypotheses :n:`{+ @ident}`.
    in :cite:`FullReduction`. If Coq is running in native code, it can be
    typically two to five times faster than :tacn:`vm_compute`. Note however that the
    compilation cost is higher, so it is worth using only for intensive
-   computations.
+   computations. According to the configuration, this tactic can default to
```

Comment:
```suggestion
   computations. With this configuration, this tactic can default to
```

"According to the configuration" doesn't sound very good.  Not sure if my suggestion means what you intend.

Reply:
Not quite. "Depending on" would fit better the intended meaning. I guess "either" should also be added before "default".